src/modules/internal_combustion_engine_control/InternalCombustionEngineControl.cpp

```diff
@@ -355,31 +355,42 @@ int InternalCombustionEngineControl::print_usage(const char *reason)
 	PRINT_MODULE_DESCRIPTION(
 		R"DESCR_STR(
 ### Description
+		
 The module controls internal combustion engine (ICE) features including:
-ignition (on/off),throttle and choke level, starter engine delay, and user request.
+ignition (on/off), throttle and choke level, starter engine delay, and user request.
 The module publishes [InternalCombustionEngineControl.msg](../msg_docs/InternalCombustionEngineControl.md).
-The architecture is as shown below.:
+		
+The architecture is as shown below:
 ![Architecture](../../assets/diagrams/ice_control_diagram.png)
 
 ### Enabling
+
 This feature is not enabled by default needs to be configured in the
 build target for your board together with the rpm capture driver:
+
+```
 CONFIG_MODULES_INTERNAL_COMBUSTION_ENGINE_CONTROL=y
 CONFIG_DRIVERS_RPM_CAPTURE=y
+```
 
 Additionally, to enable the module:
-- set [ICE_EN](https://docs.px4.io/main/en/advanced_config/parameter_reference.html#ICE_EN)
+- Set [ICE_EN](../advanced_config/parameter_reference.md#ICE_EN)
 to true and adjust the other module parameters ICE_ according to your needs.
-- set [RPM_CAP_ENABLE](https://docs.px4.io/main/en/advanced_config/parameter_reference.html#RPM_CAP_ENABLE) to true.
+- Set [RPM_CAP_ENABLE](../advanced_config/parameter_reference.md#RPM_CAP_ENABLE) to true.
 
 ### Implementation
+
 The ICE is implemented with a (4) state machine:
+
 ![Architecture](../../assets/diagrams/ice_control_state_machine.png)
+
```

Comment:
This line of empty space is needed or the following text wraps the image. If lines are not allowed then we'll need to add a line break via HTML.